### ISSUE TITLE
Query group information if we don't have it

### ISSUE THREAD

**Issue by tgalal** [tgalal/wazapp#44]
Query group information (name,icon,members)

**Comment by tgalal** [comment]
Now group info is partially queried (subject,owner,creation,subject_t,subject owner) and reflected only in chat view.  Waiting for issue #72 to reflect in conversation window.

**Comment by joaquimrocha** [comment]
Does this mean that other people in a group will show up with their name instead of their identifier? thanks in advance.

**Comment by soryuuha** [comment]
Nope, this issue only refers at creating/query group chat that you have joined. just like official client, it will auto-create group chat you've joined previously on the new phone. Wazapp yet to have this.
